source/install/desktop-app-changelog.rst

```diff
@@ -18,7 +18,7 @@ Release v5.4
 
 **Release Day: June 19, 2023**
 
-**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
```

Comment:
I don't have context on why we want to remove the "latest" link. Other changes in the PR look good to me.

Reply:
@amyblais might be okay for latest release but seems like historically folks have forgot to update this link when next release comes out so less error prone to point at release directly.

Reply:
Makes sense, I think we can leave the "latest" link here, and I'll add to the release checklist to update the link when a new release is out.

Reply:
If the submitter is ok with it, we can leave the "latest" link here, and I'll add to the release checklist to update the link when a new release is out.

Reply:
If the submitter is ok with it, we can leave the "latest" link here, and I'll add to the release checklist to update the link when a new release is out.

Reply:
@amyblais sounds good updated